### DISCUSSION TITLE
AAC+ support

### DISCUSSION THREAD

**Discussion by zdzisius** [schreibfaul1/ESP32-audioI2S#513] [Q&A]
Hello,

I have noticed that there is an update (v 3.0.0) to this fantastic library which supporting aac+ format now. Actually I have a question here, is it possible to have a good quality of sound for the aac+ internet radio? I use esp32-wroover + pcm5102a. The problem is the quality is really low - mono and 47k. Is it related to the version of esp32 or maybe something else?

here is the console log:

23:47:47.606 -> info        Connect to new host: "http://n01b-e2.revma.ihrhls.com/zc193?rj-ttl=5&rj-tok=AAABhzSG4OIAt5VOkprNl830cA"
23:47:47.652 -> info        buffers freed, free Heap: 178272 bytes
23:47:47.837 -> info        Connection has been established in 189 ms, free Heap: 178272 bytes
23:47:48.104 -> icyurl      http://www.iheartradio.com
23:47:48.136 -> info        icy-name: KOST 103.5 Los Angeles
23:47:48.136 -> station     KOST 103.5 Los Angeles
23:47:48.136 -> info        AACDecoder has been initialized, free Heap: 156536 bytes , free stack 5668 DWORDs
23:47:48.136 -> lasthost    http://n01b-e2.revma.ihrhls.com/zc193?rj-ttl=5&rj-tok=AAABhzSG4OIAt5VOkprNl830cA
23:47:48.136 -> info        stream ready
23:47:48.136 -> info        syncword found at pos 0
23:47:48.136 -> info        Channels: 1
23:47:48.169 -> info        SampleRate: 22050
23:47:48.169 -> info        BitsPerSample: 16
23:47:48.169 -> info        BitRate: 47889
23:47:48.169 -> info        AAC HeaderFormat: ADTS
23:47:48.169 -> info        AAC Codec: MPEG-2 LowComplexity
23:47:48.169 -> info        VBR recognized, audioFileDuration is estimated
23:47:49.169 -> info        slow stream, dropouts are possible
23:47:49.373 -> info        StreamTitle='title="THE BOYS OF SUMMER",artist="Don Henley",url="
23:47:49.420 -> streamtitle title="THE BOYS OF SUMMER",artist="Don Henley",url="

Thanks in advance for any comments,
regards

**Comment by schreibfaul1** [comment]
It is difficult to retrofit 'Parametric Stereo' into existing decoders. AACP is therefore considered mono for the time being.
